data/src/test/java/org/apache/iceberg/data/GenericAppenderHelper.java

```diff
@@ -68,8 +70,21 @@ public void appendToTable(DataFile... dataFiles) {
     append.commit();
   }
 
+  public void appendToTable(DataFile... dataFiles) {
+    appendToTable(null, dataFiles);
+  }
+
   public void appendToTable(List<Record> records) throws IOException {
-    appendToTable(null, records);
+    appendToTable(null, null, records);
```

Comment:
nit: this can be `appendToTable(null, records)`, right?

Reply:
Currently it won't work because we have two methods:
`appendToTable(String branch, List<Record> records) ` 
and 
`appendToTable(StructLike partition, List<Record> records)`

Unless I rearrange the order of `branch` and place it after `records`, but I prefer to leave it as it is currently, at the front.